Day8_ObjectiveC_Ass.txt

```diff
@@ -0,0 +1,370 @@
+
+Objective c assignment
+
+1)
+Ans a)
+#import <Foundation/Foundation.h>
+@interface SampleClass:NSObject
+/* method declaration */
+- (int)sumOfTwoNum;
+@end
+@implementation SampleClass
+- (int)sumOfTwoNum: (int)number1 num2: (int)number2 {
+// Your code here!
+return number1 +number2;
+}
+@end
+int main(void){
+NSAutoreleasePool* pool = [[NSAutoreleasePool alloc] init];
+/* local variable definition */
+SampleClass *sampleClass = [[SampleClass alloc]init];
+/* calling a method to print log */
+int a=[sampleClass sumOfTwoNum:6 num2:5 ];
+NSLog(@"The sum of two numbers is %i", a);
+return 0;
+}
+
+
+
+
+
+
+
+
+
+
+Ans b) #import <Foundation/Foundation.h>
+
+@interface SampleClass:NSObject
+/* method declaration */
+- (int) sumOfEven:(NSArray*)numArr;
+@end
+@implementation SampleClass
+- (int) sumOfEven:(NSArray*)numArr {
```

Comment:
- there is uneven spacing in methods
- this should be look like this - `- (int)sumOfEven:(NSArray *)numArr {`